docs/configuration.rst

```diff
@@ -124,10 +124,10 @@ These should be defined in your ``settings.py`` file.
   Default: ``True``
 * ``AXES_USERNAME_FORM_FIELD``: the name of the form field that contains your
   users usernames. Default: ``username``
-* ``AXES_USERNAME_CALLABLE``: A callable function that takes a request object
-  and returns the username. If empty, axes just fetches it from the POST body
+* ``AXES_USERNAME_CALLABLE``: A callable function that takes a request object and a credentials dictionary
```

Comment:
"credentials dictionary" is not very clearly defined. we should be more explicit.

Reply:
I changed the documentation (did incorporate the text from django docs, but also changed the wording to better fit inside a "wrapping sentence")